### PR TITLE
Pendant: Added image attribution to readme.txt

### PR DESCRIPTION
**Pull request by pbking**
This adds image attribution for all of the images found in the patterns for Pendant.

### CODE REVIEW

File: pendant/readme.txt [hunk 1]
```diff
@@ -42,3 +42,39 @@ This program is distributed in the hope that it will be useful,
...
+This theme bundles the following third-party resources:
+
+MET Museum
+License : Public Domain
+Source  : https://www.metmuseum.org/art/collection/search/243039
+          https://www.metmuseum.org/art/collection/search/248813
+          https://www.metmuseum.org/art/collection/search/452273
+	  https://www.metmuseum.org/art/collection/search/256970
+	  https://www.metmuseum.org/art/collection/search/236772
+	  https://www.metmuseum.org/art/collection/search/242952
```
Comment (madhusudhand) [inline pendant/readme.txt]: last 3 are not used in the theme. 
Also, for first 3 png versions are used in the theme. we could clean jpeg images from assets.

File: pendant/readme.txt [hunk 2]
```diff
@@ -42,3 +42,39 @@ This program is distributed in the hope that it will be useful,
...
+
+Smithsonian
+License : Creative Commons CC0
+Source  : https://www.si.edu/object/wristwatch-hour-angle-lindbergh-king-collection:nasm_A20040287020
```
Comment (madhusudhand) [inline pendant/readme.txt]: I am guessing this is for `a-watch`, but image source seems to be different.

File: pendant/readme.txt [hunk 3]
```diff
@@ -42,3 +42,39 @@ This program is distributed in the hope that it will be useful,
...
+
+PxHere
+License : Creative Commons CC0
+Source  : https://pxhere.com/en/photo/775017
+          https://pxhere.com/pt/photo/41666
```
Comment (madhusudhand) [inline pendant/readme.txt]: I don't see these two images being used in the theme.

Comment (MaggieCabrera) [reply pendant/readme.txt]: We only need to credit the images we are including in the assets files, the rest can be used in dotcom as a headstart content without the need for credit

**Comment by pbking** [comment]
@beafialho we seem to be missing links for a few of the resources.  I poked around and tried to find them but to no avail.  Can you help us out with these last few?

We don't seem to have a link for this image:
image

Or this one (though it looks like a [similar](https://www.metmuseum.org/art/collection/search/248813) one noted from the MET):
image

Or this one:
image

Or this one:
image

Also as @madhusudhand noted the .jpeg version of a few assets have been removed.

These resources seem to have been used in the figma design but aren't being used in any patterns so I've removed from the list.  As @MaggieCabrera mentioned if they only need attribution if they are included in the theme, not necessarily if they are used in the demo site or elsewhere.

https://www.metmuseum.org/art/collection/search/248813
https://www.metmuseum.org/art/collection/search/256970
https://www.metmuseum.org/art/collection/search/236772
https://www.metmuseum.org/art/collection/search/242952
https://www.si.edu/object/wristwatch-hour-angle-lindbergh-king-collection:nasm_A20040287020
https://pxhere.com/en/photo/775017
https://pxhere.com/pt/photo/41666
https://stocksnap.io/photo/people-girl-O6MBP1UB5X

**Comment by beafialho** [comment]
Sure, thanks for letting me know. Here are the correct image links:

https://stocksnap.io/photo/sketch-designer-4PMI1YMWDY
https://www.metmuseum.org/art/collection/search/248813
https://stocksnap.io/photo/people-girl-O6MBP1UB5X
https://pxhere.com/en/photo/1629996

The last two are zoomed in in the designs.

**Pull request merged**